### DISCUSSION TITLE
如何在频道中发送卡片信息？

### DISCUSSION THREAD

**Discussion by 2018haha** [Mrs4s/go-cqhttp#1488] [General]
在频道中发送xml信息无法解析，手机端不显示，电脑端提示该消息只支持手机端查看。希望能够支持频道xml

**Comment by 2018haha** [comment]
xml 中的 value 部分, 记得 html 实体化处理后, 再打加入到 CQ 码中 请问这句话是什么意思

**Comment by moxisuki** [comment]
同问
